BLSQ/iaso
# IA-3143: Update planning default settings

Related JIRA tickets : [IA-3143](https://bluesquare.atlassian.net/browse/IA-3143)

## Self proofreading checklist



## Changes
Updated planning settings by linking it to **Planning project**, **Equipment/Pop/Social mob./Microplans form**, fixed the start and end date with **Published status** and applied the planning to Health facility level instead of district!


## How to test

From setuper folder, launch `python3 setuper.py` to create data and open Planning page: [http://localhost:8081/dashboard/planning/list/](http://localhost:8081/dashboard/planning/list/) . By open the settings for the planning, you should see the changes described above.

## Print screen / video

[Coda-2.webm](https://github.com/user-attachments/assets/c90a0dee-d3d3-44f2-9cd4-e5fb548148f4)


[IA-3143]: https://bluesquare.atlassian.net/browse/IA-3143?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
